addons/website/static/src/snippets/s_hr/001.scss

```diff
@@ -0,0 +1,15 @@
+
+.s_hr[data-vcss='001'] {
+    line-height: 0;
+    hr {
+        padding: 0;
+        border: 0;
+        border-top-color: inherit;
+        margin: 0;
+    }
+    // Style
+    .s_hr_dashed { border-top-style: dashed; }
+    .s_hr_double { border-top-style: double; }
+    .s_hr_dotted { border-top-style: dotted; }
+    .s_hr_solid  { border-top-style: solid;  }
```

Comment:
@rdeodoo In fact, since you are increasing the CSS counter for a reason, I will remove those as well, less CSS :+1: Don't worry, I am doing it :)